src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx

```diff
@@ -1571,16 +1571,6 @@
     <value>An XEvent session named {0} already exists</value>
     <comment>.
  Parameters: 0 - sessionName (String) </comment>
-  </data>

```

Comment:
we should also remove these from the xlf file, right?

Reply:
that is correct